chain/jsonrpc/src/lib.rs

```diff
@@ -248,7 +248,7 @@ impl JsonRpcHandler {
     }
 
     async fn query(&self, params: Option<Value>) -> Result<Value, RpcError> {
-        let (path, data) = parse_params::<(String, String)>(params)?;
+        let (path, data, is_final) = parse_params::<(String, String, bool)>(params)?;
```

Comment:
This is a breaking change to the API, right? This API is going out of our hands and introducing it this way we just keep breaking things further. Besides, booleans are rarely a good idea for APIs as we ask people to send something that is e.g. `["account/frol.near", "", false]` (meh...).

Reply:
What do you suggest we do instead? My understanding is that the raw API is rarely used directly and people will most likely use it through nearlib, which means we can make it backward compatible by providing a default value.

Reply:
nearlib is a JS library, and I assume some may rely on the raw API as they simply implement it in another language.

I suggest creating a separate endpoint (e.g. `query-finalized` / `query-final` / ...) and internally, we can reuse the `query` method by adding a function argument to it.